blender/arm/material/make_inst.py

```diff
@@ -14,7 +14,7 @@ def inst_pos(con, vert):
         vert.write('    croty * srotz * srotx + sroty * crotx, -crotz * srotx, -sroty * srotz * srotx + croty * crotx')
         vert.write(');')
         vert.write('spos.xyz = mirot * spos.xyz;')
-        if((con.data['name'] == 'mesh' or 'translucent') and vert.contains('wnormal')):
+        if con.data['name'] == 'mesh' or con.data['name'] == 'translucent' and vert.contains('wnormal'):
```

Comment:
If I'm not mistaken `and` binds stronger than `or`, so as long as you don't really want to change the functionality of this if statement you should bring back the parentheses. The old `or 'translucent'` part was of course wrong, good catch :)

Reply:
got it